api/resources/views/vendor/mail/html/subcopy.blade.php

```diff
@@ -0,0 +1,7 @@
+<table class="subcopy" width="100%" cellpadding="0" cellspacing="0">
```

Comment:
Do we need to export all those blade files? If there is an update, we'll need to update all manually.
Nothing is changed I guess to this file